common/src/main/java/com/wynntils/features/combat/InvertKeybindsFeature.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright © Wynntils 2021-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.combat;
+
+import com.mojang.blaze3d.platform.InputConstants;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.consumers.features.properties.StartDisabled;
+import com.wynntils.core.persisted.Persisted;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.Config;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.mc.event.KeyMappingEvent;
+import com.wynntils.models.items.WynnItem;
+import com.wynntils.models.items.properties.GearTypeItemProperty;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.wynn.InventoryUtils;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import net.minecraft.client.KeyMapping;
+import net.minecraft.client.Options;
+import net.neoforged.bus.api.SubscribeEvent;
+
+@StartDisabled
+@ConfigCategory(Category.COMBAT)
+public class InvertKeybindsFeature extends Feature {
```

Comment:
Maybe `InvertAttackKeybindFeature` instead? Otherwise the name suggests a very sweeping functionality.